server/tailsql/static/script.js

```diff
@@ -1,11 +1,25 @@
 (() => {
     const query = document.getElementById('query');
+    const qButton = document.getElementById('send-query');
     const dlButton = document.getElementById("dl-button");
-    const qform = document.getElementById("qform");
+    const qform = document.getElementById('qform');
     const output = document.getElementById('output');
     const origin = document.location.origin;
     const sources = document.getElementById('sources');
 
+    function hasQuery() {
+        return query.value.trim() != "";
+    }
+
+    query.addEventListener("keydown", (evt) => {
+        if (evt.shiftKey && evt.code == "Enter") {
```

Comment:
It's slightly preferred to use `event.key` because that accounts for differences in keyboard layout / remapped keyboards etc. Probably less likely to misfire for the enter key, but you never know?

Reply:
TIL, thanks!